migrations/13_SCHEMES_GenericScheme.js

```diff
@@ -1,12 +1,48 @@
 /* global artifacts */
 /* eslint no-undef: "error" */
 
+const { getGenesisProtocolData } = require('../src/helpers/genesisProtocolHelper')(artifacts)
+
+const DxGenericScheme = artifacts.require('DxGenericScheme')
+const DxAvatar = artifacts.require('DxAvatar')
+const DxController = artifacts.require('DxController')
+
+const registerScheme = require('./helpers/registerScheme')
+
 module.exports = async function (deployer) {
-  console.log('TODO: Deploy DxSchemeRegistrar that inherits from SchemeRegistrar')
-  console.log('Configure SchemeRegistrar')
-  // TODO: Check permissions OLD_dao_migration.js
-  console.log('controller.registerScheme(_schemes[i], _params[i], _permissions[i],address(_avatar));')
+  const dxAvatar = await DxAvatar.deployed()
+  const dxController = await DxController.deployed()
+
+  console.log('Deploy DxGenericScheme that inherits from GenericScheme')
+  const dxGenericScheme = await deployer.deploy(DxGenericScheme)
+
+  console.log('Configure DxGenericScheme')
+
+  const {
+    genesisProtocolParamsHash,
+    genesisProtocolAddress
+  } = await getGenesisProtocolData()
+
+  // TODO: get locally deployed Dx address
+  const dutchXContractAddress = '0x039fb002d21c1c5eeb400612aef3d64d49eb0d94'
```

Comment:
We shouldn't hardcode

Reply:
Of course we shouldn't . Need a dx instance. Have you decided if we are deploying real DX contract or a mock?

Reply:
An instance, load from the dependency (u have the network.json), or last resource would be a config file. This is what I mean.